clang/lib/Basic/Cuda.cpp

```diff
@@ -86,7 +88,7 @@ static const CudaArchToStringMap arch_names[] = {
     // clang-format off
     {CudaArch::UNUSED, "", ""},
     SM2(20, "compute_20"), SM2(21, "compute_20"), // Fermi
-    SM(30), SM(32), SM(35), SM(37),  // Kepler
+    SM(30), SM3(32, "compute_32"), SM(35), SM(37),  // Kepler
```

Comment:
Nit. We don't really need SM3 here. For one-off we could Just use `{CudaArch::SM_32_, "sm_32" , "compute_32"}}`

Reply:
Didn't do it originally becausei tm ake the formatting a little awkward, but I'll change it.